### ISSUE TITLE
Can't sync a repository after renaming

### ISSUE THREAD

**Issue by Callidon** [lemurheavy/coveralls-public#899]
Hello,

I just rename one of my repositories ([link here](https://github.com/Callidon/augustus)) on Github and I wanted to resync it on Coveralls. So, I went on my old's repos settings and I clicked on "Resync with github ", but I got this error :
![capture](https://cloud.githubusercontent.com/assets/5773072/21563745/50aaa500-ce85-11e6-935c-5ab10a70903e.png)
The url of this page was : https://coveralls.io/repos/137285/get_new_name

Do you have a fix for this ?

**Comment by ryan-williams** [comment]
I re-ran the Travis build above, https://travis-ci.org/hammerlab/path-utils/jobs/218316822, after deleting the out-of-date-named repo from coveralls, and it seems like things are working again, though it's a shame that I am missing my coverage history now!

**tl;dr: don't add new repo names via https://coveralls.io/repos/new, go to the repo's settings and resync with GitHub**

**Comment by akaIDIOT** [comment]
Being hit by this as well. In the process of trying to fix it though, I 'deleted' the repo on coveralls to try and make it show up on the linked repo overview so I could make it resurface, but now the original has been deleted and I can't add the new one as it never shows up on a resync...

Is there a workaround path for this?

**Comment by jupe** [comment]
I've exactly same issue. Can't get it work anymore at all after renaming.. What can I do ?
